envi/archs/arm/emu.py

```diff
@@ -120,15 +131,47 @@ def c1101(flags):
         c1101,
         ]
 
-class ArmEmulator(ArmModule, ArmRegisterContext, envi.Emulator):
+top_bits_32 = [(e_bits.u_maxes[4] ^ ((e_bits.u_maxes[4]>>x))) for x in range(4*8)]
+
+LSB_FMT = [0, 'B', '<H', 0, '<I', 0, 0, 0, '<Q',]
+MSB_FMT = [0, 'B', '>H', 0, '>I', 0, 0, 0, '>Q',]
+LSB_FMT_SIGNED = [0, 'b', '<h', 0, '<i', 0, 0, 0, '<q',]
+MSB_FMT_SIGNED = [0, 'b', '>h', 0, '>i', 0, 0, 0, '>q',]
+
+# SIMD support
+OP_F16 = 1
+OP_F32 = 2
+OP_F64 = 3
+OP_S32 = 4
+OP_U32 = 5
+
+ifs_first_F32 = (IFS_F32_S32 | IFS_F32_U32 | IFS_F32_F64 | IFS_F32_F16)
```

Comment:
let's see if they still exist when i'm done implementing i_vcvt.

Reply:
this may change at a later date, but for now there are comments in const.py and emu.py.

also some fixes, since simdflags actually ended up being an index holder.  may change to actual flags later after some contemplation on performance impact.  leaving "simdflags" as the name for now unless you object.  i'm happy to change now and then again later if we choose to move back to flags (which was obviously the original idea).